### DISCUSSION TITLE
read  and validate an entity "table" in dxf

### DISCUSSION THREAD

**Discussion by nquinterno** [mozman/ezdxf#684] [Q&A]
hi,

its posible to read and validate  an entity "table" that was inserted in a dxf?. I didnt find anything related to that in EZDXF documentation. 

Thaks
![consulta_hithub_tablas](https://user-images.githubusercontent.com/22855208/168396779-4cb24874-fb09-40f6-bd35-fef00132c7be.png)

**Comment by mozman** [comment]
There is no support whatsoever for the `ACAD_TABLE` entity.

**Reply by varunkaarthik** [reply]
Hi @mozman,

Could you suggest an alternative way to read information from an entity table? (with or without ezdxf)

Thanks
